strtoint/strtoint.go

```diff
@@ -0,0 +1,12 @@
+package strtoint
+
+import "strconv"
+
+func MyStrToInt(s string) (int, error) {
+	i, err := strconv.Atoi(s)
```

Comment:
```suggestion
	return strconv.Atoi(s)
```

Reply:
In this case, It will no pass `Out of range test` because besides of error, it will also return the maximum possible number `9223372036854775807,` instead of `0`

> i = 9223372036854775807 
> err = "strconv.Atoi: parsing "999999999999999999999999999999999999": value out of range"

Reply:
This is not a problem. When you call a function that return `error` - you should check an error firstly. And, if error is not nil - you shouldn't  use the first value.

Reply:
Few times in the code of different repositories I saw ignore the error and still use the result of the function. Something like `res, _ := somefunc()` (sure it's not a good code-style, but I saw it a few times).
For example, `MyStrToInt3` https://github.com/VSKut/go-temp/pull/10/commits/092aefc750dc6b86cc9f73417126b5df56e89344#diff-8aaf57f411b4048135d195de5fa6b6f7R7 in case of `Out of range` will return `0, error`, but in your fix `MyStrToInt`, `MyStrToInt2` returns `maxNumOfType, error`.
In my opinion in order to unification they should return the same result.
But anyway updated https://github.com/VSKut/go-temp/pull/10/commits/9f9b32d80dfe66ac9f7935afd6bdb715edcc50ae https://github.com/VSKut/go-temp/pull/10/commits/b592da9857478c3267a16d23f9e88d6e71d4c0f6 because this is just a homework :)